portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/DB2SQLTransformerLogicTest.java

```diff
@@ -66,7 +66,7 @@ public void testReplaceCastText() {
 	public void testReplaceConcat() {
 		Assert.assertEquals(
 			"select * from Foo where foo LIKE CAST(bar AS VARCHAR(2000)) " +
-				"CONCAT COALESCE(CAST(? AS VARCHAR(2000)),'')",
+				"CONCAT ?",
```

Comment:
@yuhai Please change the commit message to something like "After reverting, the ? after CONCAT is not replaced. It's legal usage in DB2 whether the ? after CONCAT is replaced or not, so keep it untouched and adjust the test case".

Reply:
@dantewang, I just changed the commit message and pushed force it. Thanks.